<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FAPConfDialog</class>
 <widget class="QDialog" name="FAPConfDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="dRPCLabel">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Discord RPC</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="dRPCCheckBox">
     <property name="text">
      <string>Enabled</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="dRPCIDLabel">
     <property name="text">
      <string>Application ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="dRPCIDLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="dRPCTokenLabel">
     <property name="text">
      <string>User Token</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="dRPCTokenLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
